src/View/SSTemplateParser.peg

```diff
@@ -16,15 +16,6 @@ this is: framework/src/View):
 
 See the php-peg docs for more information on the parser format, and how to convert this file into SSTemplateParser.php
 
-TODO:
-    Template comments - <%-- --%>
-    $Iteration
-    Partial cache blocks
-    i18n - we dont support then deprecated _t() or sprintf(_t()) methods; or the new <% t %> block yet
-    Add with and loop blocks
-    Add Up and Top
-    More error detection?
```

Comment:
Most (or all even?) of this is already done, and we don't keep todo comments in code anyway.